modules/moa-installing-prerequisites.adoc

```diff
@@ -24,7 +24,12 @@ This user will have Programmatic access enabled and the AdministratorAccess poli
 ====
 +
 . Install and configure the AWS CLI.
-.. Follow the AWS documentation to install the link:https://aws.amazon.com/cli/[aws-cli] for your operating system.
+.. Follow the AWS documentation to install and configure the link:https://aws.amazon.com/cli/[aws-cli] for your operating system.
+.. Optional: Set a default AWS region. `moactl` evaluates regions in the following priority order:
++
+... The region specified when running a `moactl` command with the `--region` flag.
+... The region set in the `AWS_DEFAULT_REGION` environment variable. See the link:https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html[AWS documentation for more details].
+... The default region set in your AWS configuration file. See the link:https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html#cli-configure-quickstart-config[AWS documentation for more details].
```

Comment:
```suggestion
... The default region set in your AWS configuration file. See link:https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html#cli-configure-quickstart-config[Quick configuration with `aws configure`] in the AWS documentation.
```

Reply:
Yes, I understand now, I worked with Kevin on it. This has been done. I'll ask Kevin to merge the changes. Thanks for your great suggestions!